### PR TITLE
Korean Translation for 3.4.5

### PR DESCRIPTION
**Pull request by kimtj12**
### What does it do?

Korean Translation Update for 3.4.5

### Why is it needed?

Korean Translation for 3.4.5

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/strapi-admin/admin/src/translations/ko.json [hunk 1]
```diff
@@ -14,147 +18,298 @@
...
-  "components.AutoReloadBlocker.description": "다음 명령어 중 하나를 사용하여 Strapi를 실행합니다:",
-  "components.AutoReloadBlocker.header": "이 플러그인은 리로드 기능이 필요합니다.",
-  "components.ErrorBoundary.title": "에러가 발생했습니다.",
+  "app.utils.publish": "공개",
+  "app.utils.select-all": "전체 선택",
+  "app.utils.unpublish": "비공개",
+  "component.Input.error.validation.integer": "숫자(integer) 값을 제공해주세요",
+  "components.AutoReloadBlocker.description": "다음 명령어 중 하나를 사용하여 Strapi를 실행해보세요:",
+  "components.AutoReloadBlocker.header": "이 플러그인은 Reload 기능이 필요합니다.",
+  "components.ErrorBoundary.title": "에러가 발생했습니다... ㅠㅠ",
```
Comment (iicdii) [inline packages/strapi-admin/admin/src/translations/ko.json:272]: I think this line is unnecessary change.

File: packages/strapi-admin/admin/src/translations/ko.json [hunk 2]
```diff
@@ -165,15 +320,28 @@
   "components.Wysiwyg.selectOptions.H5": "제목 H5",
   "components.Wysiwyg.selectOptions.H6": "제목 H6",
   "components.Wysiwyg.selectOptions.title": "제목",
-  "components.WysiwygBottomControls.charactersIndicators": "문자 표시기",
+  "components.WysiwygBottomControls.charactersIndicators": "문자 수",
   "components.WysiwygBottomControls.fullscreen": "전체화면",
-  "components.WysiwygBottomControls.uploadFiles": "파일을 끌어 놓으세요. 혹은 클립보드에서 붙혀넣거나 {browse} 하세요.",
+  "components.WysiwygBottomControls.uploadFiles": "파일을 끌어 놓거나, 클립보드에서 붙혀넣거나, {browse} 하세요",
```
Comment (iicdii) [inline packages/strapi-admin/admin/src/translations/ko.json:325]: "클립보드에서 붙여 넣거나," is the correct grammar. origin translation was wrong, too. Can you correct it?

File: packages/strapi-admin/admin/src/translations/ko.json [hunk 3]
```diff
@@ -165,15 +320,28 @@
...
   "components.popUpWarning.message": "삭제하시겠습니까?",
-  "components.popUpWarning.title": "확인",
+  "components.popUpWarning.title": "확인해주세요",
+  "form.button.continue": "계속",
   "form.button.done": "확인",
-  "notification.error": "에러가 발생했습니다.",
+  "form.button.finish": "완료",
+  "global.prompt.unsaved": "해당 페이지를 나가시겠습니까? 저장되지 않은 항목들은 유실됩니다",
+  "notification.contentType.relations.conflict": "콘텐츠 타입에 충돌이 나는 릴레이션(relation)이 있습니다",
+  "notification.error": "에러가 발생했습니다 ㅜㅜ",
```
Comment (iicdii) [inline packages/strapi-admin/admin/src/translations/ko.json:336]: Here again, I think "에러가 발생했습니다." is enough.

**Review by iicdii — CHANGES_REQUESTED** [review]

**Comment by alexandrebodin** [comment]
Hi @kimtj12 anyway you can check the few commentson your PR ? would love to merge it asap :)

**Comment by alexandrebodin** [comment]
Hi, closing for inactivity. Feel free to ping me if you want to take it up sometime in the future :)

**Pull request closed without merging**